src/fsharp/MethodCalls.fs

```diff
@@ -68,7 +68,7 @@ type CalledArg =
       NameOpt: Ident option
       CalledArgumentType : TType }
 
-let CalledArg(pos,isParamArray,optArgInfo,callerInfoInfo,isOutArg,nameOpt,reflArgInfo,calledArgTy) =
+let GetCalledArg(pos,isParamArray,optArgInfo,callerInfoInfo,isOutArg,nameOpt,reflArgInfo,calledArgTy) =
```

Comment:
Is this because Fable doesn't let you have a type and a function with the same name in the same module? thanks